### DISCUSSION TITLE
Where did the Unicode names come from?

### DISCUSSION THREAD

**Discussion by ctrlcctrlv** [kovidgoyal/kitty#4312] [Q&A]
It's giving strange names to codepoints in the PUA, like "Nodejs"

![image](https://user-images.githubusercontent.com/838783/144497736-4c379bf0-7a3e-4c31-957c-601d82d4652f.png)

**Comment by page-down** [comment]
Are these from `nerd-fonts-glyphs.txt` ?

```text
e718 dev nodejs small
e719 dev nodejs
f898 mdi nodejs
```

**Comment by kovidgoyal** [comment]
Yes those are from the nerd font icons.
